docs/enterprise-setup/implementation-guide.md

```diff
@@ -122,45 +166,40 @@ See the [following guide](/access-management/sso-providers/azure-entra-id) on ho
 </TabItem>
 </Tabs>
 
-To configure basic auth (deploy without SSO), remove the entire `auth:` section from your airbyte.yml config file. You will authenticate with the instance admin user and password included in the your `airbyte.yml`.
+To configure basic auth (deploy without SSO), exclude the entire `auth:` section from your airbyte.yml config file. You will authenticate with the instance admin user and password included in your `airbyte.yml`.
 
-To modify auth configurations after Airbyte is installed, you will need to redeploy Airbyte with the additional environment variable `KEYCLOAK_RESET_REALM=TRUE`. As this also resets the list of Airbyte users and permissions, please use this with caution.
+To modify auth configurations after Airbyte is installed, you will need to `helm upgrade` Airbyte with the additional environment variable `--set keycloak-setup.env_vars.KEYCLOAK_RESET_REALM=true`. As this also resets the list of Airbyte users and permissions, please use this with caution.
```

Comment:
Do we have a callout template of some sort that we could highlight this section with?

Reply:
No, since this is in a collapsible section. But I did move it up to be slightly more visible. We'll see if customers continue to miss this.